omod/src/test/java/org/openmrs/module/datafilter/extension/html/ContentTest.java

```diff
@@ -21,17 +21,16 @@
 	
 	@Test
 	public void addLocationNameLine() {
-		Content content = new Content(Arrays.asList("L1", "L2"), new ArrayList<String>());
+		Content content = new Content(Arrays.asList("L'1", "L2"), new ArrayList<String>());
 		
 		String expected = "<style>.listItemBoxCustom {width: 460px;padding: 2px;border: 1px solid lightgray;float: left;background-color: #EFEFEF;overflow-x: scroll;height: 200px;}</style>"
 		        + "<td valign='top'>Location</td>" + "<td valign='top'><div id='locationStrings' class='listItemBoxCustom'>"
 		        + "<span class='listItem'>"
-		        + "<input type='checkbox' name='locationStrings' id='locationStrings.L1' value='L1'><label for='locationStrings.L1'>L1</label>"
+		        + "<input type='checkbox' name='locationStrings' id='locationStrings.L&#39;1' value='L&#39;1'><label for='locationStrings.L&#39;1'>L&#39;1</label>"
```

Comment:
That's ok for the sake of the assert, but we need to make sure that at app runtime all works as expected. Can you confirm?

Reply:
Yes, I tested it on my environment.